fasjsonclient/openapi_client/exceptions.py

```diff
@@ -0,0 +1,120 @@
+# coding: utf-8
+
+"""
+    Fedora Account Service JSON API
+
+    fajson rest api  # noqa: E501
+
+    The version of the OpenAPI document: 0.0.1
+    Generated by: https://openapi-generator.tech
+"""
+
```

Comment:
We should probably make the exceptions in this module more accessible for users, because they may need them in their code. Maybe import them in a top-level module?

Reply:
so everything that is within the openapi_client module is generated by the openapi tooling.. I avoided changing anything there but I am starting to wonder it is worth it

Reply:
Yeah... I wasn't thinking of changing things there, just re-importing those exceptions in the top-level module so users have a direct access to them.